### DISCUSSION TITLE
[4.x]: Money Fields - Conditional Display

### DISCUSSION THREAD

**Discussion by ambidustrious** [craftcms/cms#11560] [Ideas] [answered]
### Description
Money Fields aren't available as fields when setting up conditional display logic in the control panel

### Steps to reproduce

1. Create 2 Money Fields
2. Set the second Money field to display only when the first Money field isn't empty

### Expected behavior

I would expect to see the Money fields in the list of conditional Logic fields

### Actual behavior
All Money fields are missing from this list

### Craft CMS version

4.1.1

### PHP version

8

### Operating system and version

OSX Monterrey

### Database type and version

MySQL

### Installed plugins and versions

-
Neo
Blitz
SuperTable
Freeform
Image Optimize
SEOmatic

**Comment by brandonkelly** [comment, marked as answer]
Craft 4.2.0 is out now, with support for referencing Money fields in conditions ✨
